vscode-extension/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "relay-lsp-client",
+  "version": "1.0.0",
+  "description": "Relay LSP Client for VSCode",
+  "main": "./out/extension.js",
+  "activationEvents": [
+    "onLanguage:plaintext",
+    "onLanguage:javascript",
+    "onLanguage:javascriptreact",
+    "onLanguage:typescript",
+    "onLanguage:typescriptreact"
+  ],
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "Relay",
+      "properties": {
+        "relay.enabled": {
+          "scope": "window",
+          "type": "boolean",
+          "default": true,
+          "description": "Is relay enabled"
+        },
```

Comment:
Do we need this? People can just enable/disable the extension I think? Is there some other value?

Reply:
yeah vscode can disable extensions per workspace now
<img width="520" alt="image" src="https://user-images.githubusercontent.com/6754223/161678127-82b0e151-5cc9-46b9-b6f2-76d23e5ee668.png">